tests/integration_tests/services/test_sagemaker.py

```diff
@@ -1,17 +1,12 @@
-import pytest
-
 from ai21 import SageMaker
-from tests.integration_tests.skip_helpers import should_skip_studio_integration_tests
 
 
-@pytest.mark.skipif(should_skip_studio_integration_tests(), reason="No key supplied for AI21 Studio. Skipping.")
```

Comment:
Can you please add a test where we set an env variable of `AI21_API_KEY` to validates that it still works?

Reply:
👍